src/monitors.ts

```diff
@@ -46,23 +68,23 @@ export function buildMonitorParams(monitor: Monitor, cloudFormationStackId: stri
     `service:${service}`,
   ];
 
-  if (checkIfRecommendedMonitor(serverlessMonitorId)) {
-    let criticalThreshold = SERVERLESS_MONITORS[serverlessMonitorId].threshold;
+  if (isRecommendedMonitor(serverlessMonitorId, recommendedMonitors)) {
+    let criticalThreshold = recommendedMonitors[serverlessMonitorId].threshold;
+
     if (monitorParams.options) {
       if (monitorParams.options.thresholds) {
         if (monitorParams.options.thresholds.critical) {
```

Comment:
Two comments here
1) The last check is `if (monitorParams.options.thresholds.critical)` which will be false when the value for critical threshold is 0 in which case the custom threshold will not get picked up. This check should instead be `if (monitorParams.options.thresholds.critical !== undefined)`

2) This being a typescript file, instead of having 3 nested if conditions, this can be simplified to...
```
if (monitorParams.options?.thresholds?.critical !== undefined) {
```